decidim-participatory_processes/lib/decidim/participatory_processes/admin_engine.rb

```diff
@@ -150,6 +150,26 @@ class AdminEngine < ::Rails::Engine
 
       initializer "decidim_participatory_processes_admin.process_components_menu" do
         Decidim.menu :admin_participatory_process_components_menu do |menu|
+          current_participatory_space.components.each do |component|
+            caption = translated_attribute(component.name)
+            if component.primary_stat.present?
+              caption += content_tag(:span, component.primary_stat, class: component.primary_stat.zero? ? "component-counter component-counter--off" : "component-counter")
```

Comment:
Can you add some styling to the `.component-counter`? If not, it's difficult to differentiate the numbers between this counter and the component name

Reply:
Thanks for Suggestion, it has been fixed as part of [bcba67a](https://github.com/decidim/decidim/pull/11654/commits/bcba67a640612446417a510dcacd08c24968a5f0).

Reply:
Ok, we can iterate on that

Reply:
Not very clear for me, on what we can iterate on ? I just moved the Counter to the right. I could make a circle or something around it.

Reply:
We have this now...
![image](https://github.com/decidim/decidim/assets/105683/2a30113c-289c-40f2-ae7c-091dcc5f3045)

Reply:
Too much blue. I left a comment with a suggestion in the CSS file